birdhouse/optional-components/stac-populator/default.env

```diff
@@ -0,0 +1 @@
+export STAC_ASSET_GENERATOR_TIMEOUT=30
```

Comment:
This is missing the variable registration for expansion in templates. 
See example: 
https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/optional-components/secure-data-proxy/default.env#L9-L17

Also, this component should probably have the `components/stac` as a dependency.
See example: 
https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/optional-components/secure-data-proxy/default.env#L30-L36